### ISSUE TITLE
touch screen support for Chrome

### ISSUE THREAD

**Issue by forman** [phosphorjs/phosphor#154]
Panels can't be dragged in Chrome browsers using a touch screen. Observed with the [DockPanel example](http://phosphorjs.github.io/examples/dockpanel/) on a Dell XPS laptop running Windows 10. With Firefox is works fine.

**Comment by jasongrout** [comment]
FYI, Chrome 55 has PointerEvents, which may make this easier when it comes out: https://www.chromestatus.com/feature/4504699138998272

**Comment by sccolbert** [comment]
@forman Thanks. This is a known issue. The DockPanel is really targeted at desktop users with a mouse. Once the PointerEvents spec is implemented everywhere, we'll move over to that. It's not a priority for me at the moment to support touch events.

**Comment by stefan505** [comment]
Any progress on this?

**Comment by jasongrout** [comment]
FYI, it looks like firefox is still implementing PointerEvents and Safari has not implemented it.

**Comment by stefan505** [comment]
Yeah, I saw that thanks. Just wondered if any work has started on it for the supported platforms.
